### PR TITLE
Hidden Password Prompt added and Readme updated

### PR DESCRIPTION
**Pull request by barry-1928**
1. The hidden password functionality has been added.
2. The README.md file has been updated to incorporate the changes made.

### CODE REVIEW

**Review by Anirudh-Swaminathan — CHANGES_REQUESTED** [review]
Hey there.
Thanks for the pull request.
I had tried this solution before, but there are a few problems from the UX point of view to look into -
1. If the -p tag does nothing, it is better to not have it at all.
2. The "hanging" -p could either be missed, leading to an error, or the -p could be followed by the users password, which also leads to an error that is not caught.

Expected ideal behaviour would be to have the command as follows
```python
python scrape_it.py -r <your_roll_number> -p <your_password>
```
and when implemented looks something like this
```python
python scrape_it.py -r blah -p *********
```
The stars represent the characters of the password. It could also be blank, just like passwords are entered in Linux too.

In short, we have 2 options.
1. The **easy** way out is to remove -p completely and then use only -r as an argument. Enter the password inside the program(using `getpass`)
2. The other way is to modify the argument -p so as to be able to input password in an invisible manner.

**Comment by barry-1928** [comment]
I am not sure if keeping track of what has been already entered on terminal and then making a particular portion of text hidden is possible(at least from what I searched). Tried searching a lot but couldn't get any clue.

I have used the first option as you mentioned above(by removing -p argument) and made a commit. We generally do enter passwords on a separate line, even in general Linux commands(Like in git push origin <branch_name>, we do enter the authentication information in separate lines). So, I don't think it would create any UX issue anymore. Please review the changes.

Thank you.

**Comment by Anirudh-Swaminathan** [comment]
Hey. Thanks for the Pull Request again.
I have merged the proposed changes for now.
The second option proposed by me is indeed difficult to achieve for now, especially with just Python. So I'll leave that issue for another day for now.

Thanks again.

**Comment by barry-1928** [comment]
Hey!

It was my pleasure to contribute to your project. Good Job done! 👍 

Thank You!
